cmake/third_party/azure_sdk.cmake

```diff
@@ -0,0 +1,87 @@
+include(ExternalProject)
+include(FetchContent)
```

Comment:
We can remove this, since we don't use it.

Reply:
removed in https://github.com/wiredtiger/wiredtiger/pull/8702/commits/e48383f1ca180dc7222bcd3c57585e590dfcd30f